src/LeanCode.ContractsGenerator.Tests/LeanCode.ContractsGenerator.Tests.csproj

```diff
@@ -6,28 +6,32 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
-  <Target Name="PackDependencies" BeforeTargets="Build" Condition="'$(IsInnerBuild)' != 'true'">
-    <MSBuild Projects="../LeanCode.Contracts/LeanCode.Contracts.csproj" Properties="Configuration=Debug" Targets="Restore" />
-    <MSBuild Projects="../LeanCode.Contracts/LeanCode.Contracts.csproj" Targets="Build;Pack" />
+  <ItemGroup>
+    <FxToCompile Include="$(TargetFrameworks)" />
+  </ItemGroup>
+
+  <Target Name="PackDependenciesForTests" BeforeTargets="AfterBuild">
+    <Message Text="Packing the LeanCode.Contracts" Importance="high" />
+    <MSBuild
+      StopOnFirstFailure="true"
+      Projects="../LeanCode.Contracts/LeanCode.Contracts.csproj"
+      Properties="Configuration=Debug;TargetFramework=net6.0;TargetFrameworks=net6.0;Version=9.9.9.9-internal"
+      Targets="Restore;Build;Pack" />
   </Target>
 
+  <ItemGroup>
+    <Compile Include="../../examples/notifications/**/*.cs" />
+  </ItemGroup>
+
   <ItemGroup>
     <Content Include="../../examples/**/*">
       <Link>examples/%(RecursiveDir)%(Filename)%(Extension)</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="ExampleBased/Notifications/Examples/**/*">
-      <Link>examples/notifications/%(RecursiveDir)%(Filename)%(Extension)</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
     <Content Include="../LeanCode.Contracts/bin/Debug/LeanCode.Contracts.*.nupkg">
       <Link>examples/%(Filename)%(Extension)</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="../LeanCode.Contracts/bin/Release/LeanCode.Contracts.*.nupkg">
```

Comment:
Does nuget find this `9.9.9.9-internal` package when running tests with release configuration?

Reply:
Yeah, the `Pack` target run above forces the `Debug` configuration now, so even though the tests are run in release, the contracts are built in debug.

Reply:
idk if you checked this also but I remember I had to run `dotnet nuget locals all --clear` between checking  `dotnet test --configuration Debug` and `dotnet test --configuration Release` as nuget could cache this package between those test runs causing Release to pass locally and on github action to fail.

Reply:
Checked. Works